airbyte-integrations/connectors/source-cockroachdb/build.gradle

```diff
@@ -14,7 +14,7 @@ application {
 }
 
 dependencies {
-    implementation 'org.postgresql:postgresql:42.6.0'
+    implementation 'org.postgresql:postgresql:42.7.2'
```

Comment:
Does the vulnerability list `source-cockroachdb` too ? It should filter on certified destinations i thought

Reply:
I have removed the change for source-cockroachdb